packages/usa-input-mask/src/content/usa-input-mask-date-short.json

```diff
@@ -0,0 +1,10 @@
+{
+  "hint": "For example, 01/01/1970",
```

Comment:
It is unclear if this example is telling the user to input MM-DD-YYYY or DD-MM-YYYY. Could the example be something that more clearly describes the format? It might not matter for our examples, but I want to make sure we are guiding users to write helpful hints.

Reply:
Changed this to 01/31/1970, also did this in unit test to keep it consistent.